examples/p4est_3d_dgsem/elixir_euler_free_stream_boundaries.jl

```diff
@@ -0,0 +1,60 @@
+
+using Downloads: download
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations3D(1.4)
+
+initial_condition = initial_condition_constant
+
+d = 2
+solver = DGSEM(polydeg = d, surface_flux = flux_lax_friedrichs)
```

Comment:
```suggestion
solver = DGSEM(polydeg = 2, surface_flux = flux_lax_friedrichs)
```

Why `polydeg = 2` instead of the common `polydeg = 3`?